src/release/installer/mac/build.xml

```diff
@@ -110,6 +110,10 @@
       <option value="-Xmx512m"/>
       <option value="-Dapple.laf.useScreenMenuBar=true"/>
 
+      <!-- Work around conflict with local JAI extensions installed in ~/Library/Java/Extensions -->
+      <!-- (system extension  MRJToolkit used by launcher to open the default browser) -->
+      <option value="-Djava.ext.dirs=${env.JAVA_HOME}/jre/lib/ext:/System/Library/Java/Extensions"/>
```

Comment:
I've had JAI show up in `/System/Library/Java/Extensions`, especially on older OS X machines that have been upgraded a few times, so this is a partial fix at best.